lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -630,6 +630,7 @@ export class UserAgentApplication {
      *
      */
     acquireTokenSilent(userRequest: AuthenticationParameters): Promise<AuthResponse> {
+        this.logger.info("AcquireTokenSilent has been called");
```

Comment:
I think logs like this at the start of APIs should be verbose.

Reply:
Or at-least highlight what is called, `request` specifics if it helps with log analysis, not just the API name. Depending on PII settings ofcourse.

Reply:
Noted. Changed in commit.